create_derivatives/routines.py

```diff
@@ -129,9 +133,133 @@ def ocr_pdf(self):
 
 
 class ManifestMaker:
-    # TO DO: make manifests
+    """Creates a IIIF presentation manifest version 3 from JP2 files.
+
+    Creates manifest directory in bag's data directory and then creates manifest.
+
+    Returns:
+        A tuple containing human-readable message along with list of bag identifiers.
+        Exceptions are raised for errors along the way.
+    """
+
+    def __init__(self):
+        server_url = settings.IMAGESERVER_URL
+        self.resource_url = "{}/iiif/3/".format(server_url)
+        self.fac = ManifestFactory()
+        self.fac.set_base_prezi_uri("{}/manifests/".format(server_url))
+        self.fac.set_base_image_uri(self.resource_url)
+        self.fac.set_debug(settings.PREZI_DEBUG)
+        self.upgrader = Upgrader()
+        self.manifest_dir = None
+        self.jp2_path = None
+        self.jp2_files = None
```

Comment:
You don't need this.

Reply:
Think I thought I needed these based on some some incorrect logic in the tests, but just didn't realize what I was doing at the time.

Thanks!